frontend/src/i18n/constants.ts

```diff
@@ -112,6 +112,7 @@ export const KEY = {
   duration: 'duration',
   publication: 'publication',
   saksdokument: 'saksdokument',
+  common_required: 'påkrevd',
```

Comment:
Ta en titt på  docstring øverst
```suggestion
  common_required: 'common_required',
```